src/routes/api/update/+server.ts

```diff
@@ -0,0 +1,26 @@
+import { UpdateEvent, update_events } from '$lib/stores/update'
+
+export function GET() {
+    const event = new UpdateEvent()
+    update_events.push(event)
+
+    const stream = new ReadableStream({
+        start(controller) {
+            event.on('update', () => {
+                controller.enqueue('event: update\ndata: {"message": "State updated"}\n\n')
+            })
+        },
+        cancel() {
+            const index = update_events.indexOf(event)
```

Comment:
hisssssss

Reply:
use map with uuid i guess??

events_map[uuid] = event

and when delete delete events_map[uuid]